src/main/java/com/github/stefvanschie/inventoryframework/GuiItem.java

```diff
@@ -139,6 +148,14 @@ public void callAction(@NotNull InventoryClickEvent event) {
     public void setAction(@NotNull Consumer<InventoryClickEvent> action) {
         this.action = action;
     }
+    
+    public List<Object> getProperties(){
+        return properties;
+    }
+    
+    public void setProperties(List<Object> properties){
+        this.properties = properties;
+    }
```

Comment:
Please add javadoc comments to these two methods as well as nullability and contract annotations (`@NotNull` for the parameter in `setProperties` and for the return type in `getProperties` and a `@Contract(pure = true)` to the `getProperties` method).